CMakeLists.txt

```diff
@@ -602,6 +602,13 @@ if(HPX_WITH_THREAD_STEALING_COUNTS)
   hpx_add_config_define(HPX_HAVE_THREAD_STEALING_COUNTS)
 endif()
 
+hpx_option(HPX_WITH_COROUTINE_COUNTS BOOL
```

Comment:
nitpick: should we name this `..._COUNTERS` instead?

Reply:
yes

Reply:
Counters was my first choice but there was prior art with counts... There are actually examples of both. `HPX_WITH_IO_COUNTERS`, `HPX_WITH_THREAD_STEALING_COUNTS`. I'm absolutely okay with changing it to counters.

Reply:
I changed the name to `HPX_WITH_COROUTINE_COUNTERS`.